front-end/pom.xml

```diff
@@ -1,10 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>fr.maaxow.bring-breakfast</groupId>
-		<artifactId>parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
+    <groupId>fr.maaxow.bring-breakfast</groupId>
+    <artifactId>bring-breakfast-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
```

Comment:
If you use 4  spaces indentations, can you apply this in all the file ?
I prefer tab indentations, but you commit all file with 4 spaces, so i'm ok.

Reply:
maybe think about adding a .editorconfig file in your project to handle this stuff.